src/web/tools/InteractionManager.ts

```diff
@@ -97,10 +99,13 @@ export default class InteractionManager {
 
   public shouldHandlerBeCancelledBy(
     _handler: GestureHandler,
-    _otherHandler: GestureHandler
+    otherHandler: GestureHandler
   ): boolean {
-    //TODO: Implement logic
-    return false;
+    return (
+      otherHandler instanceof NativeViewGestureHandler &&
+      otherHandler.getLastSentState() === State.ACTIVE &&
+      otherHandler.getState() === State.ACTIVE
```

Comment:
Is there a reason for checking both, `state` and `lastSentState`?

Reply:
After some testing I can tell that there isn't. Initially I used that to check whether scroll was already active, or just got activated, but it seems that it isn't required anymore.